Update Copy/Paste details - group widget functionality (#6286)
DataDog/documentation: content/en/graphing/widgets/_index.md

Additional options are available for [timeseries widgets][1].

> ## Copy / Paste Widgets

Reviewer: ```suggestion
## Copy / paste widgets
```
According to our guidelines, h2 headings are sentence case

Widgets can be copied on [Dashboards][2], [Notebooks][3], [APM Service][4], and the [APM resource][5] page by using `Ctrl + C` (`Cmd + C` for Mac), or by selecting the share icon and choosing "Copy". 

The copied widgets can be pasted within Datadog by using `Ctrl + V` (`Cmd + V` for Mac) on:

* **Dashboards**: Adds a new widget positioned under your mouse cursor.
* **Notebooks**: Adds a new cell at the end of the notebook.

> When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.

Reviewer: ```suggestion
You can also paste the widget into your favorite chat program that displays link previews (like Slack or Microsoft Teams). This displays a snapshot image of your graph along with a direct link to your widget.
```

> ### Groups of Widgets

Reviewer: ```suggestion
### Groups of widgets
```

> Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 

Reviewer: ```suggestion
Timeboard group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 
```

> *Note: When pasting to Screenboards or Notebooks, individual widgets within the group will be pasted*

Reviewer: ```suggestion
**Note**: When pasting graphs to screenboards or notebooks, individual widgets within the group are pasted.
```

> To copy multiple screenboard widgets (edit mode only), `shift + click` on multiple widgets and using `Ctrl + C` (`Cmd + C` for Mac). 

Reviewer: ```suggestion
To copy multiple screenboard widgets (edit mode only), `shift + click` on the widgets and use `Ctrl + C` (`Cmd + C` for Mac). 
```

> *Note: This only works when sharing within Datadog, this will not generate a preview image outside of Datadog*

Reviewer: ```suggestion
**Note**: This only works when sharing within Datadog. It does generate a preview image outside of Datadog.
```

Additional options are available for [timeseries widgets][1].

## Copy / Paste Widgets

Widgets can be copied on [Dashboards][2], [Notebooks][3], [APM Service][4], and the [APM resource][5] page by using `Ctrl + C` (`Cmd + C` for Mac), or by selecting the share icon and choosing "Copy". 

The copied widgets can be pasted within Datadog by using `Ctrl + V` (`Cmd + V` for Mac) on:

* **Dashboards**: Adds a new widget positioned under your mouse cursor.
* **Notebooks**: Adds a new cell at the end of the notebook.

When pasting into your favorite chat program that displays link previews (like Slack or Microsoft Teams) a snapshot image of your graph will be displayed along with a direct link to your widget.

### Groups of Widgets

Timeboard Group widgets can be copied by hovering over the group widget area and using `Ctrl + C` (`Cmd + C` for Mac) or by selecting the share icon and choosing "Copy". 

> *Note*: When pasting to Screenboards or Notebooks, individual widgets within the group will be pasted

Reviewer: ```suggestion
**Note**: When pasting graphs to screenboards or notebooks, individual widgets within the group are pasted.
```

To copy multiple screenboard widgets (edit mode only), `shift + click` on multiple widgets and using `Ctrl + C` (`Cmd + C` for Mac). 

> *Note*: This only works when sharing within Datadog, this will not generate a preview image outside of Datadog

Reviewer: ```suggestion
**Note**: This only works when sharing within Datadog. It does generate a preview image outside of Datadog.
```

Author: @ruthnaebeck this is untrue It should be "It does NOT"
